symengine/tests/basic/test_relationals.cpp

```diff
@@ -112,10 +115,20 @@ TEST_CASE("Canonicalization", "[Relationals]")
 TEST_CASE("Eq", "[Relationals]")
 {
     RCP<const Symbol> x = symbol("x");
+    RCP<const Symbol> y = symbol("y");
 
     RCP<const Basic> a = Eq(x);
     RCP<const Basic> b = Eq(x, zero);
     CHECK(eq(*a, *b));
+
+    a = add(x, y);
+    b = add(a, zero);
+    CHECK(eq(*Eq(a, b), *boolTrue));
+
+    b = add(a, real_double(0.0));
+    CHECK(eq(*Eq(a, b), *boolTrue));
+    CHECK(eq(*Eq(sub(b, y), x), *boolTrue));
+    CHECK(eq(*Eq(add(x, real_double(0.0)), x), *boolTrue));
```

Comment:
These tests shouldn't be in test relationals. They should be in `test_arit`

Reply:
Added them there as well. I'd leave these as a compatibility check for original issue.